src/ui/app/components/transaction.jsx

```diff
@@ -495,9 +495,17 @@ const getTimestamp = (date) => {
 };
 
 const getAddressCredentials = (address) => {
-  const cmlAddress = Loader.Cardano.Address.from_bech32(address);
-  return [cmlAddress.payment_cred()?.to_cbor_hex(), cmlAddress.staking_cred()?.to_cbor_hex()];
-}
+  try {
+    const cmlAddress = Loader.Cardano.Address.from_bech32(address);
+    return [
+      cmlAddress.payment_cred()?.to_cbor_hex(),
+      cmlAddress.staking_cred()?.to_cbor_hex(),
+    ];
+  } catch (error) {
+    console.error(error);
+    return [null, null];
```

Comment:
Does code that call this function handle nulls?

Reply:
the only time this gets checked is in `matchesAnyCredential` which compares the owners own payment/staking cred, with this one, so comparison with `'someKindOfString' === null` should still return the right value for us